Tests/Request/RequestBodyParamConverter20Test.php

```diff
@@ -182,8 +190,8 @@ public function testApplyWithSerializerContextOptionsForSymfonySerializer()
     {
         $this->serializer = $this->getMock('Symfony\Component\Serializer\SerializerInterface', array('serialize', 'deserialize'));
         $this->converter = new RequestBodyParamConverter20($this->serializer);
+        $this->fillContainer($this->converter);
         $requestBody = '{"name": "Post 1", "body": "This is a blog post"}';
-
         $options = array(
```

Comment:
Is this important ? Because the new context class supports it just with attributes.